internal/config/config.go

```diff
@@ -16,7 +16,7 @@ const (
 
 const (
 	appData        = "AppData"
-	configFilePath = "bk.yaml"
+	configFilePath = ".bk.yaml"
```

Comment:
When using the global config file, we don't want the file to be hidden. So this will need some further changes I think.

Or possible just revert this change, since it looks hard coded below for the "local" config